docs/abcd_rooparametrichist_tutorial/rooparametrichist_exercise.md

```diff
@@ -0,0 +1,462 @@
+param_Bkg_B_norm = RooAddition("bkg_B"+"_norm","Total Number of events from background in control region B",process_B_region_bins)
+
+#Here we import the the parametric histogram and the normalization in our workspace
+getattr(ws, "import")(param_hist_B_region, RooFit.Rename("bkg_B"))
+getattr(ws, "import")(param_Bkg_B_norm, RooFit.Rename("bkg_B"+"_norm"),RooFit.RecycleConflictNodes())
+
+#Define the parametric histogram for control region C.
+#The RooParametricHist is initalized giving as input the observable, the RooArgList of the bins previously built and a template TH1F. 
+param_hist_C_region = RooParametricHist("bkg_C", "Background PDF in C region",variable_z,process_C_region_bins,histC_pr)
+
+#Here we define the total normalization for the RooparametricHist in the C region
+param_Bkg_C_norm = RooAddition("bkg_C"+"_norm","Total Number of events from background in control region C",process_C_region_bins)
+
+#Here we import the the parametric histogram and the normalization in our workspace
+getattr(ws, "import")(param_hist_C_region, RooFit.Rename("bkg_C"))
+getattr(ws, "import")(param_Bkg_C_norm, RooFit.Rename("bkg_C"+"_norm"),RooFit.RecycleConflictNodes())
+
+#Define the parametric histogram for control region D.
+param_hist_D_region = RooParametricHist("bkg_D", "Background PDF in D region",variable_z,process_D_region_bins,histD_pr)
+
+#Here we define the total normalization for the RooparametricHist in the D region
+param_Bkg_D_norm = RooAddition("bkg_D"+"_norm","Total Number of events from background in control region D",process_D_region_bins)
+
+#Here we import the the parametric histogram and the normalization in our workspace
+getattr(ws, "import")(param_hist_D_region, RooFit.Rename("bkg_D"))
+getattr(ws, "import")(param_Bkg_D_norm, RooFit.Rename("bkg_D"+"_norm"),RooFit.RecycleConflictNodes())
+
+```
+</details>
+
+For the signal region A, create a ```RooParametricHist``` with each bin made from a ```RooFormulaVar``` relating the C,D,B regions to A via the ABCD formula. 
+
+<details>
+<summary> Create RooParametricHist for SR background template  </summary>
+
+```
+#Relate the signal region (A) to control region B via transfer factors
+#Define RooArgList for expected yields in region A after applying transfer factors
+process_AB_region_bins = RooArgList()
+#Define list for transfer factors
+TF_list = []
+#Define list for expected yields in region A after applying transfer factors
+process_AB_region_bins_list = []
+
+#Compute per-bin transfer factor
+#Loop over the bins of the transfering region B, and compute the transfer factors as C/D
+for i in range(1,histB_pr.GetNbinsX()+1):
+    #Define transfer factor as a RooFormulaVar. Use the method .obj for RooWorkSpace to retrieve the yield for a given bin and region
+    TF_i = RooFormulaVar("TF"+str(i),"Transfer factor C/D bin " + str(i),"(@0/@1)",RooArgList(ws.obj("Bkg_C_region_bin_"+str(i)) , ws.obj("Bkg_D_region_bin_"+str(i)) ))
```

Comment:
This part I certainly agree with! I think my comment wasnt very clear. I didn't mean not to use one per bin, I was just wondering about the use of the Transfer factors TF_i. 

They are an intermediate which is not really necessary, (but also completely fine to include!). The full equation is "A_i = B_i C_i / D_i" and this can be written in a number of different ways, including the way that you've done defining "TF_i = C_i / D_i". I am used to thinking of it in the first way, so I find it slightly clearer -- which I'm trying to consider in this pedagogical context, but I'm not sure if that's just me. Other people may find things clearer thinking with the TF_i intermediary.

Reply:
Now i see, thanks for the clarification ! As you prefer, if we talk about a "per-bin transfer factor method" then maybe might help to define TF_i in the code, but i agree in general is not necessary